packages/oats/src/generate-types.ts

```diff
@@ -227,24 +228,25 @@ export function run(options: Options) {
 
   function generateQueryType(
     op: string,
+    opts: Options,
```

Comment:
do you need to pass `opts` around here? The functions are already in a closure that has it